README.rst

```diff
@@ -22,8 +22,8 @@ You can test the documentation locally through simply running
 
     make html
```

Comment:
also the shortlog does not have a verb, could you add one so that it still makes sense :) ?